src/corehost/common/pal.h

```diff
@@ -211,7 +211,9 @@ namespace pal
     bool getenv(const char_t* name, string_t* recv);
     bool get_default_servicing_directory(string_t* recv);
     bool get_local_dotnet_dir(string_t* recv);
-    bool get_global_dotnet_dir(string_t* recv);
+    //As long as linux uses the path to find the global location of 
```

Comment:
I would suggest we rephrase this to say that since, on Linux, we determine global location by enuemrating the location where dotnet is present on path, there could be multiple such locations. We should also say that on Windows, it will only be one.